src/screens/Details.js

```diff
@@ -13,8 +14,7 @@ const template = `
 `;
 
 export const DetailsScreen = async ({ renderOn }) => {
-    const params = new URLSearchParams(window.location.search);
-    const searchID = params.get('id');
+    const { id: searchID } = getQueryParams();
```

Comment:
It's weird that the variable name is searchID, it should be assetId

Reply:
that's not my code, i just refactored it to use my method, but ok i will change that